protocol/contracts/lpp/src/contract/rewards.rs

```diff
@@ -112,7 +112,7 @@ mod test {
             deps.as_mut().storage,
             Config::new(
                 TheCurrency::TICKER.into(),
-                1000u64.into(),
+                Code::unchecked(1000u64),
```

Comment:
This might be on me for not following it when I came, but I feel like we should stick to one magic numbers for a given purpose.

Reply:
do not get what you say

Reply:
this is a test code, after all